app/src/common/metric-names.ts

```diff
@@ -0,0 +1,10 @@
+const server = 'jenkins-for-jira.forge.server';
+
+export const metricResolverEmitter = {
+	fetchJenkinsEventHandlerUrl: `${server}.fetchServer.emitted`,
+	connectJenkinsServer: `${server}.createServer.emitted`,
+	updateJenkinsServer: `${server}.updateServer.emitted`,
+	getAllJenkinsServers: `${server}.getAllServers.emitted`,
+	getJenkinsServerWithSecret: `${server}.getServer.emitted`,
```

Comment:
same as above, should we have "withSecret" in metrics name or remove it from the key?